CONTRIBUTING.md

```
@@ -0,0 +1,168 @@
+* Include *reproduction steps* that someone else can follow to recreate the bug or error on their own.
+* Please be ready to provide that information if maintainers ask for it.
+
+Once it's filed:
+
+* The project team will [label the issue](#label-issues).
+* A team member will try reproducing the issue with your steps. If there are no repro steps or no obvious way to reproduce the issue, the team will ask you for those steps and mark the issue as `needs-repro`. Bugs with the `needs-repro` tag will not be addressed until they are reproduced.
+* If the team can reproduce the issue, it will be marked `needs-fix`, as well as possibly other tags (such as `critical`).
+* If you or the maintainers don't respond to an issue for 30 days, the [issue will be closed](#clean-up-issues-and-prs). If you want to come back to it, reply (once, please), and we'll reopen the existing issue. Please avoid filing new issues as extensions of one you already made.
+* `critical` issues may be left open, depending on perceived immediacy and severity, even past the 30-day deadline.
+
+ ## Request Support
```

Markdownlint (0.37.0)

Headings must start at the beginning of the line

(MD023, heading-start-left)